src/packages/client/src/generation/TSClient/Args.ts

```diff
@@ -18,18 +17,8 @@ export class ArgsType implements Generatable {
     const { action, args } = this
     const { name } = this.model
 
-    const singular = name
-    const plural = pluralize(name)
```

Comment:
Oh so I think that was the last place we were using pluralize, maybe we can check to remove it completely @timsuchanek